Jenkinsfile

```diff
@@ -107,8 +135,12 @@ pipeline {
       steps {
         sh 'make setup-git'
         sh 'git fetch --all'
-        unstash 'psammead-publishes'
-        sh 'make talos'
+        script {
+          if (publishesStashed) {
+            unstash 'psammead-publishes'
+          }
+        }
+        sh "npm run talos ${params.TALOS_PACKAGES}"
```

Comment:
This is quite insecure since anyone can pass in values into the TALOS_PACKAGES field and then have them passed to run on Jenkins. eg `; npx randompackagehere`.

Reply:
I'd suggest running a short input validation script that checks all values passed in are namespaced under `@bbc/` and that errors with a helpful message if anyone passes in something else.

Reply:
Means we can’t use this to bump other deps though :( like styled components